packages/openapi-framework/src/types.ts

```diff
@@ -15,6 +15,34 @@ export {
   OpenAPIErrorTransformer
 };
 
+export class ConsoleDebugAdapterLogger implements Logger {
+  /**
+   * `console.debug` is just an alias for `.log()`, and we want debug logging to be optional.
+   * This class delegates to `console` and overrides `.debug()` to be a no-op.
+   * @param message
```

Comment:
can you remove these 2 `@param` comments?

Reply:
sure